monero-GUI-guide.md

```diff
@@ -287,28 +293,52 @@ The `Sign/verify` tab provides tools for signing a message or file with your pri
-**(12) Change location:** Use a GUI to enter a non-default path to the blockchain.    
-**(13) Show advanced:** Check this box to show advanced options (Startup flags or Node login).    
-**(14) Startup flags/Node login:** When using a local node this will be where you enter additional command line options. When using a remote node this will be where you can enter a username and password in case authentication is required.    
-**(15) Custom decorations:** Check this box to show the Monero custom decorations.    
-**(16) Log level:** Change the verbosity of the debug logs.    
-**(17) Log categories:** Add specific categories to the debug logs.    
-**(18) Wallet creation height:** Change the block height that a wallet rescan will go back to. Click `Save` for changes to take effect.    
+### 6.0.1. Wallet
+
+![settings-wallet](media/black_settings-wallet.png)
+
+### 6.0.2a. Local Node
+
+![settings_local-node](media/black_settings-node-local_node.png)
+
+**(1) Start/Stop Local Node:** Depending on its current state, either start or stop the local node.    
+**(2) Blockchain location:** Manually enter a non-default path to the blockchain.    
+**(3) Startup flags:** When using a local node, this will be where you enter additional command line options.    
+**(4) Bootstrap Address:** Enter the hostname or IP address of the bootstrap remote node. See section [8.1 Bootstrap nodes](#8.1-bootstrap-nodes) for a brief explanation of what is a bootstrap node    
+**(5) Bootstrap Port:** Enter the port of the bootstrap remote node.
+
+### 6.0.2b. Remote Node
+
+Use a remote node, do not download the blockchain. Check the ['About remote nodes' section](#8-about-remote-nodes) of this guide.
+
+![settings_remote-node](media/black_settings-node-remote_node.png)
+
+**(1) Address:** Enter the hostname or IP address of the remote node.    
+**(2) Port:** Enter the port of the remote node.    
+**(3) Daemon Username:**  enter a username in case authentication to the remote node is required.
+**(4) Daemon Password:**  enter a password in case authentication to the remote node is required.
+
+### 6.0.3. Log
+
+![settings-log](media/black_settings-log.png)
+
+**(1) Log level:** Change the verbosity of the debug logs.    
+**(2) Log categories:** Add specific categories to the debug logs.    
+**(3) Daemon log:** Real time output of the log.    
+**(4) Command line:** Interact with the daemon.
+
+### 6.0.4. Info
+
+![settings-info](media/black_settings-info.png)
+
+**(1) GUI Version:** Version of the GUI wallet installed.    
+**(2) Embedded Monero Version:** Version of the embedded daemon in use.    
+**(3) Wallet path:** Where the wallet is located on your computer.    
+**(4) Wallet creation height:** Change the block height that a wallet rescan will go back to.    
+**(5) Wallet log path:** Where the logs for this wallet will be saved.    
+**(6) Copy to clipboard:** Copy ll these informations to the clipboard.    
```

Comment:
copy ll?

Reply:
it supposed to be `all` :)
Thanks, all fixed